bonus-a-cloud/.mvn/wrapper/maven-wrapper.properties

```diff
@@ -0,0 +1,2 @@
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.8.4/apache-maven-3.8.4-bin.zip
```

Comment:
I see other sections have 3.8.1, maybe we should align ?

Reply:
Yes, Maybe one of the projects have been generated after the quarkus maven wrapper has been updated